diff --git a/build.bat b/build.bat
index 50ad683..ddf5e4d 100644
--- a/build.bat
+++ b/build.bat
@@ -365,7 +365,7 @@ IF /I "%~2" == "Win32" (
 
 PUSHD "%BIN_DIR%"
 
-SET "PCKG_NAME=%NAME%.%MPCHC_VER%.%ARCH%"
+SET "PCKG_NAME=%NAME%.%MPCHC_VER%.%ARCH%.ICL13"
 IF DEFINED MPCHC_LITE (SET "PCKG_NAME=%PCKG_NAME%.Lite")
 IF /I "%COMPILER%" == "VS2012" (SET "PCKG_NAME=%PCKG_NAME%.%COMPILER%")
 
diff --git a/distrib/mpc-hc_setup.iss b/distrib/mpc-hc_setup.iss
index aa1cc36..620ee16 100644
--- a/distrib/mpc-hc_setup.iss
+++ b/distrib/mpc-hc_setup.iss
@@ -41,7 +41,7 @@
     #define localize = "true"
   #endif
 #endif
-#define sse_required
+#define sse2_required
 
 
 ; From now on you shouldn't need to change anything
@@ -81,6 +81,8 @@
   #define OutFilename   = app_name + "." + app_ver + ".x86"
 #endif
 
+#define OutFilename     = OutFilename + ".ICL13"
+
 #ifnexist bindir + "\" + mpchc_exe
   #error Compile MPC-HC first
 #endif
@@ -109,6 +111,9 @@
 #ifdef MPCHC_LITE
   #define FullAppNameVer = FullAppNameVer + " " + "Lite"
 #endif
+
+#define FullAppNameVer = FullAppNameVer + " " + "ICL13"
+
 #ifdef x64Build
   #define FullAppNameVer = FullAppNameVer + " " + "(64-bit)"
 #endif
diff --git a/docs/Readme.txt b/docs/Readme.txt
index 8e4ed5e..5c47921 100644
--- a/docs/Readme.txt
+++ b/docs/Readme.txt
@@ -27,7 +27,7 @@ Main Features:
 
 System Requirements:
 ---------------------
-* An SSE capable CPU
+* An SSE2 capable CPU
 * Windows XP SP3, Vista, 7, 8 32-bit/64-bit
 
 
diff --git a/src/common.props b/src/common.props
index c6ba69d..57047a3 100644
--- a/src/common.props
+++ b/src/common.props
@@ -47,11 +47,14 @@
       <SubSystem>Windows</SubSystem>
       <TargetMachine Condition="'$(Platform)'=='Win32'">MachineX86</TargetMachine>
       <TargetMachine Condition="'$(Platform)'=='x64'">MachineX64</TargetMachine>
+      <WPOAsmListLocation>$(IntDir)</WPOAsmListLocation>
+      <WPOObjectFile>$(IntDir)</WPOObjectFile>
     </Link>
     <ResourceCompile>
       <Culture>0x0409</Culture>
       <PreprocessorDefinitions Condition="'$(Platform)'=='Win32'">WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(Platform)'=='x64'">_WIN64;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <SuppressStartupBanner>true</SuppressStartupBanner>
     </ResourceCompile>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug' Or '$(Configuration)'=='Debug Filter' Or '$(Configuration)'=='Debug Lite'">
@@ -73,24 +76,27 @@
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Release' Or '$(Configuration)'=='Release Filter' Or '$(Configuration)'=='Release Lite'">
     <ClCompile>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
-      <EnableEnhancedInstructionSet Condition="'$(Platform)'=='Win32'">StreamingSIMDExtensions</EnableEnhancedInstructionSet>
+      <DisableSpecificDiagnostics>597,997,13200</DisableSpecificDiagnostics>
+      <EnableFiberSafeOptimizations>true</EnableFiberSafeOptimizations>
+      <FloatingPointModel>Fast</FloatingPointModel>
       <InlineFunctionExpansion>AnySuitable</InlineFunctionExpansion>
+      <InterproceduralOptimization>SingleFile</InterproceduralOptimization>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <FavorSizeOrSpeed>Speed</FavorSizeOrSpeed>
       <OmitFramePointers Condition="'$(Platform)'=='Win32'">true</OmitFramePointers>
-      <Optimization>MaxSpeed</Optimization>
+      <Optimization>MaxSpeedHighLevel</Optimization>
       <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
       <StringPooling>true</StringPooling>
-      <WholeProgramOptimization Condition="'$(UseLTCG)'=='true'">true</WholeProgramOptimization>
+      <WholeProgramOptimization>true</WholeProgramOptimization>
     </ClCompile>
     <Lib>
-      <LinkTimeCodeGeneration Condition="'$(UseLTCG)'=='true'">true</LinkTimeCodeGeneration>
+      <LinkTimeCodeGeneration>true</LinkTimeCodeGeneration>
     </Lib>
     <Link>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <LargeAddressAware Condition="'$(Platform)'=='Win32'">true</LargeAddressAware>
-      <LinkTimeCodeGeneration Condition="'$(UseLTCG)'=='true'">UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
+      <LinkTimeCodeGeneration>UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
       <MergeSections>.rdata=.text</MergeSections>
       <OptimizeReferences>true</OptimizeReferences>
       <SetChecksum>true</SetChecksum>
diff --git a/src/filters/transform/VSFilter/installer/vsfilter_setup.iss b/src/filters/transform/VSFilter/installer/vsfilter_setup.iss
index 16d9cc7..2f9cf41 100644
--- a/src/filters/transform/VSFilter/installer/vsfilter_setup.iss
+++ b/src/filters/transform/VSFilter/installer/vsfilter_setup.iss
@@ -25,7 +25,7 @@
 #endif
 
 ; If you want to compile the 64-bit version define "x64Build" (uncomment the define below or use build.bat)
-#define sse_required
+#define sse2_required
 ;#define VS2012
 ;#define x64Build
 
@@ -58,6 +58,8 @@
   #define OutFilename   = app_name + "_" + app_version + "_x86"
 #endif
 
+#define OutFilename     = OutFilename + ".ICL13"
+
 #ifnexist bindir + "\VSFilter.dll"
   #error Compile VSFilter first
 #endif
@@ -75,6 +77,9 @@
 #if MPC_BETA_RELEASE
   #define FullAppNameVer = FullAppNameVer + " " + str(MPC_VERSION_BETA)
 #endif
+
+#define FullAppNameVer = FullAppNameVer + " " + "ICL13"
+
 #ifdef x64Build
   #define FullAppNameVer = FullAppNameVer + " " + "(64-bit)"
 #endif
diff --git a/src/platform.props b/src/platform.props
index 7da55ea..dd03fbf 100644
--- a/src/platform.props
+++ b/src/platform.props
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Configuration">
-    <PlatformToolset Condition="'$(VisualStudioVersion)'=='11.0' And '$(ANALYZE)'!='true'">v110_xp</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)'=='11.0' And '$(ANALYZE)'!='true'">Intel C++ Compiler XE 13.0</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='11.0' And '$(ANALYZE)'=='true'">v110</PlatformToolset>
   </PropertyGroup>
 </Project>
diff --git a/src/thirdparty/ffmpeg/Makefile b/src/thirdparty/ffmpeg/Makefile
index b684c3d..abe7607 100644
--- a/src/thirdparty/ffmpeg/Makefile
+++ b/src/thirdparty/ffmpeg/Makefile
@@ -58,7 +58,7 @@ ifeq ($(64BIT),yes)
 else
     TARGET_OS  = i686-w64-mingw32
     CFLAGS    += -DWIN32 -D_WIN32 -DARCH_X86_32
-    OPTFLAGS  += -m32 -march=i686 -mmmx -msse -mfpmath=sse
+    OPTFLAGS  += -m32 -march=i686 -mmmx -msse -msse2 -mfpmath=sse
     YASMFLAGS += -f win32 -m x86 -DWIN32=1 -DARCH_X86_32=1 -DARCH_X86_64=0 -DPREFIX
 endif
 
