Index: moreuuids.h
===================================================================
--- moreuuids.h	(revision 1363)
+++ moreuuids.h	(working copy)
@@ -440,6 +440,11 @@
 DEFINE_GUID(MEDIASUBTYPE_HDMVSUB,
 0x4eba53e, 0x9330, 0x436c, 0x91, 0x33, 0x55, 0x3e, 0xc8, 0x70, 0x31, 0xdc);
 
+/* Microsoft Windows SDK 2008 compile fix */
+// {34FFCBC3-D5B3-4171-9002-D4C60301697F }
+DEFINE_GUID(MEDIASUBTYPE_DVB_SUBTITLES,
+0x34FFCBC3, 0xD5B3, 0x4171, 0x90, 0x02, 0xD4, 0xC6, 0x03, 0x01, 0x69, 0x7F);
+
 #pragma pack(push, 1)
 typedef struct {
 	DWORD dwOffset;	
